doc-content/enterprise-only/getting-started/test-speed-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='testing-speed-limit-proc']
+= Testing the speed limit scenario
+
+Test the speed limit scenario using the data that you specified when you created the speed violation guided decision table.
+
+.Prerequisites
+
+* Created the `Driver_department_traffic_violations` project
```

Comment:
You have ... and period at the end of each

Reply:
Corrected throughout.